Makefile

```diff
@@ -40,7 +40,6 @@ compose:
 .PHONY: test
 test: compose
 	yarn test
-	yarn e2e
```

Comment:
I'm guessing this deletion was unintentional?

Reply:
Yes. As I thought about it, it seemed appropriate that a default "test" make target should stick to the Node 15+ requirement for building the plugin. Running `yarn e2e` directly is still available, but that would of course, put the onus on the developer to ensure Node 18+ is available for its use.

Reply:
Resolved via PM, in agreement.